pkg/operator/operator.go

```diff
@@ -22,8 +22,6 @@ import (
 const (
 	zoneLabel = "topology.kubernetes.io/zone"
```

Comment:
The `zoneLabel` constant is no longer used so the code review step of the build fails as it does not allow "dead code". Remove this line and the following commented out `regionLabel` line.